bindata/etcd/restore-pod.yaml

```diff
@@ -26,6 +26,11 @@ spec:
         env | grep ETCD | grep -v NODE
         export ETCD_NODE_PEER_URL=https://${NODE_NODE_ENVVAR_NAME_ETCD_URL_HOST}:2380
 
+        # checking if there are any fio perf file left behind that could be deleted without problems
+        if [ ! -z $(ls -A "/var/lib/etcd/etc_perf*") ]; then
```

Comment:
@claneys Hello :) 

Is it `etc_perf*` or `etcd_perf*` ?

Reply:
Oh crap, yes you're right. Thanks. I'll fix it and update the PR.

@tjungblu  @Elbehery, ok, no problem. Thanks for the tip.